lib/diagnostic.md

```diff
@@ -29,7 +41,14 @@ DOM node. (Assume the list item has a class of "listerItem").
 Give two examples of jQuery getter methods and two examples of jQuery setter methods.
 
 <!-- your answer starts here -->
+'''
+Getter: $( "li.item-a" ).parent()
+$( ".target" ).show();
```

Comment:
so $('.target') is, but show is a method to show it.  Let me know if you have any questions on this.